frontend/src/store/episode/actions.ts

```diff
@@ -42,6 +42,19 @@ const createComment = createAsyncThunk<Comment, CommentFormCreatePayload, AsyncT
   return comment;
 });
 
+const toggleCommentLike = createAsyncThunk <Comment, number, AsyncThunkConfig>
+(ActionType.LIKE_COMMENT, async (commentId, { extra, getState }) =>{
+  const { commentApi } = extra;
+  const { auth } = getState();
+  const currentComment = await commentApi.getById(commentId);
+  const checkIsLiked = currentComment.commentReactions.some((reaction) => reaction.userId === auth.user?.id);
```

Comment:
```suggestion
  const isLiked = currentComment.commentReactions.some((reaction) => reaction.userId === auth.user?.id);
```
you already have this functionality in the `checkIsLiked` helper. move this comment to the `frontend/src/helper/comment` and reuse it here